services/github/src/tc-yaml.js

```diff
@@ -199,9 +200,11 @@ class VersionOne extends TcYaml {
           prefix + payload.details['event.base.repo.branch'],
         ];
       }
-    } else if (payload.tasks_for === 'github-release') {
+    } else if (payload.tasks_for === GITHUB_TASKS_FOR.RELEASE) {
+      // Assume separate role if this is a pre-release action
+      const role = payload.action === GITHUB_RELEASE_ACTION.PRERELEASED ? 'pre-release' : 'release';
```

Comment:
Then again, technically this change is backwards incompatible anyway. To your earlier question, I don't expect this to cause much (if any) disruption. But it's definitely possible it breaks someone.

Reply:
Indeed. I think we should not change the `release` one.

Reply:
alright, since this is a breaking change, we make this role match the action of the event to be consistent: `assume:repo:github.com/<owner>/<repo>:release:<action>`